install/mysql/glpi-empty.sql

```diff
@@ -1342,7 +1342,7 @@ INSERT INTO `glpi_configs` VALUES ('174','core','notifications_ajax','0');
 INSERT INTO `glpi_configs` VALUES ('175','core','notifications_ajax_check_interval', '5');
 INSERT INTO `glpi_configs` VALUES ('176','core','notifications_ajax_sound', NULL);
 INSERT INTO `glpi_configs` VALUES ('177','core','notifications_ajax_icon_url', '/pics/glpi.png');
-INSERT INTO `glpi_configs` VALUES ('178','core','dbversion','9.3-dev');
+INSERT INTO `glpi_configs` VALUES ('178','core','dbversion','9.4.2');
```

Comment:
Not necessary

Reply:
Should I remove this change?

Reply:
Maybe should we set `FILLED AT INSTALL` as for the `version`?

Reply:
I agree.

Reply:
done